.github/workflows/stale-issue-pr.yml

```diff
@@ -1,7 +1,78 @@
 name: 'Close stale issues and PRs'
+
 on:
-  schedule:
-    - cron: '30 * * * *'
+  workflow_call:
+    inputs:
+      stale_issue_message:
+        description: 'Message to post when marking an issue as stale'
+        required: false
+        default: 'This issue has been automatically marked as stale because it has not had recent activity in the last 30 days. It will be closed in 60 days. Thank you for your contributions.'
+        type: string
+      close_issue_message:
+        description: 'Message to post when closing a stale issue'
+        required: false
+        default: 'This issue was closed because there has been no follow activity in 90 days. If you feel this was closed in error please provide evidence on the current production app in a new issue or comment in the existing issue to a maintainer. Thank you for your contributions.'
+        type: string
+      stale_issue_label:
+        description: 'Label to use when marking an issue as stale'
+        required: false
+        default: 'stale'
+        type: string
+      any_of_issue_labels:
+        description: 'Comma-separated list of labels to check for issues'
+        required: false
+        default: 'type-bug, needs-information, needs-reproduction'
```

Comment:
I see this was pre-existing, but is `type-bug` intended to be in this list? That would make this apply to all bugs, but I thought it was intended to only apply to `needs-information` and `needs-reproduction` bugs. Those two labels are only applied to bugs anyway, making them redundant right now in this config.

Reply:
```suggestion
        default: 'needs-information, needs-reproduction'
```